scss/detail.scss

```
@@ -34,133 +26,344 @@ $container-width: 900px;
-    height: 20px;
-    margin-top: 33px;
-    justify-content: center;
+  .container{
     align-items: center;
+    background-color: $white-primary;
+    display: flex;
+    flex-direction: column-reverse;
+    justify-content: center;
+    height: calc(#{$total-height} - 101px);
+    width: calc(#{$total-width} - 990px);
+    
+    .back-arrow {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      max-width: 1024px;
+      width: auto;
+      height: 20px;
+      margin-top: 33px;
+      cursor: pointer;
+    }
+    
+    .container-book {
+      flex-direction: column;
+      align-items: center;
+      height: 570px;
+      width: 336px;
+      
+      .container-img {
+        height: 337px;
+        padding: 38px 48px 0 48px;
+        width: 336px;
+    
+        .badge {
+          height: calc($badge-height + 25%);
+          padding: 18px;
+          width: calc($badge-width + -30%);
+          right: 10px;
+          top: 8px;
+        }
+      }
+      
+      .container-detail {
+        height: 233px;
+        padding: 21px;
+        width: 336px;
+        
+        .header-detail {
+          .title-book{
+            font-size: 24px;
+            line-height: 24px;
+          }
+      
+          .title-gender {
+            font-size: 16px;
+            line-height: 16px;
+          }
+        }
+      
+        .detail-form {
+          margin-top: 18px;
+      
+          .detail-item {
+            font-size: 14px;
+            line-height: 14px;
+          }
+      
+          .detail-description {
+            font-size: 14px;
+            line-height: 14px;
+          }
+        }
+      }
+    }
   }
+}
 
-  .container-book {
-    flex-direction: column;
-    align-items: center;
-    height: calc(#{$container-height} + 151px);
-    width: calc(#{$container-width} - 564px);
+@media(max-width: 853px) {
+  .container {
+    height: 768px;
+    width: 1024px;
+    display: grid;
+    justify-items: center;
+    grid-template-rows: 53px 1fr;
+    grid-template-columns: 1fr;
 
-    .container-img {
-      height: 337px;
-      padding: 38px 48px 0;
-      width: 336px;
-  
-      .badge {
-        height: calc(#{$badge-height} - 5%);
-        padding: 8px;
-        width: calc(#{$badge-width} - 5%);
-        right: 22px;
-        top: 18px;
-      }
+    .back-arrow {
+      grid-column: 1 / 1 / 3 / 2;
+      align-items: center;
+      justify-content: flex-start;
+      width: 853px;
+      margin-top: 33px;
+      padding: 0;
     }
   
-    .container-detail {
-      height: auto;
-      padding: 18px;
-      width: 100%;
+    .container-book {
+      height: 569px;
+      width: 853px;
+      align-items: start;
+      background-color: $white-secondary;
+      box-shadow: 4px 4px 10px 0 $shadow;
+      display: grid;
+      grid-template-columns: 325px 1fr;
+      grid-template-rows: 100px 1fr;
+      grid-template-areas: "detai detail"
+      "image detail";
```

Indent this rule so it's easier to visualize the template
```suggestion
      grid-template-areas: "detail detail"
                           "image detail";
```